client/dogs/FormPage.js

```diff
@@ -363,8 +364,8 @@ export default class IndexPage extends React.Component {
         color: result['priColor'] || '',
         breed: result['priBreed'] || '',
         sex: result['sex'] || '',
-        rabiesIssued: result['vacDate'] || '',
-        rabiesExpire: result['vacExp'] || '',
+        rabiesIssued: Moment(result['vacDate']).format('L') || '',
+        rabiesExpire: Moment(result['vacExp']).format('L') || '',
```

Comment:
I'd recommend lowercase `l` because it doesn’t have ugly leading zeros, at least based on my read of the docs.